packages/@react-spectrum/list/test/ListView.test.js

```diff
@@ -604,6 +610,68 @@ describe('ListView', function () {
       expect(document.activeElement).toBe(rows[rows.length - 2]);
       expect(rows[rows.length - 1]).toHaveAttribute('aria-disabled', 'true');
     });
+
+    // TODO: fix this test, it appears that the table item animation out isn't running
```

Comment:
Is this tracked with a separate issue?

Reply:
hoping someone has an idea while reviewing honestly